### ISSUE TITLE
User docs are possibly out of date

### ISSUE THREAD

**Issue by MichaelGoodale** [osekilab/CANDS#1]
Hi, this library looks quite cool! Unfortunately it looks like there's no way to build the CLI interface right now as all the relevant code is commented out in the `lib.rs` file. It seems there's no way to straightforwardly build the relevant parts from the `cli` module since when I tried to run it with the CLI related code uncommented, there was a huge number of import errors. Is this intentional or is the process of getting the CLI working not quite yet finished?

**Comment by iccazzos** [comment]
Hi, thanks for the issue!

At an earlier stage of this project, this crate was a binary crate, and we were working on the command-line interpreter.  The user docs reflect an early version of this interpreter.  But then we decided to stop working on the interpreter and instead on the actual implementation of Collins & Stabler (2016), and we turned this into a library crate.  We thought that it would be best to separate the C&S '16 implementation separate from tools that depend on it, e.g., an interpreter, or tools such as `derivck` and `derivexp` which we describe in the paper (and have yet to make public).  When we made this switch from a binary crate to a library crate, we commented out most of the interpreter code.  The library itself has grown a lot more since then, so the interpreter code is mostly obsolete and would require a major rehaul to make it work with the current library (although that could be a future project!).

Right now, the only way to use the C&S '16 implementation in this library is by importing this library and writing Rust code to use it.  I will try and make GitHub repositories for the two tools we describe in our paper (`derivck` and `derivexp`) as soon as possible.  But maybe we could start working on the interpreter again!
